packages/server/src/lib/email.js

```diff
@@ -77,7 +77,11 @@ function sendPassCode(email, passcode, httpOrigin, redirectTo) {
     );
 
     if (process.env.DEV_LOGIN_LINK && process.env.NODE_ENV === 'development') {
-        console.log(`Login link generated: \x1b[32m${href}`);
+        const BLUE = '\x1b[34m';
```

Comment:
I wasn't sure if the preference was for something like this vs using a library like chalk. We seem to have chalk installed but only as a dev dependency, so I didn't want to rely on it here, but I'm happy to update if that feels cleaner.

Reply:
🤔 Given that this execution path is specific to dev workflows, is there a way to use chalk without declaring it as a production dependency? 

In general, my preference for logging is to slowly migrate over to structured logging (we currently use `bunyan` for this, with a preconfigured base logger [exported by `lib/logging.js`](https://github.com/usdigitalresponse/usdr-gost/blob/main/packages/server/src/lib/logging.js#L16-L21)). However, given that this is specifically intended to be human-readable and not something we would try to query with a tool like Datadog, it may be best to leave the colorization as-is for now (assuming there isn't a way to use chalk here without bringing it in as a prod dependency).

Reply:
It's possible to use chalk here from devDependencies, but I think it ends up more messy than worth it. You run into a few issues you have to skirt: 

1. The airbnb base eslint config we use disallows devDependencies being used outside test and config files (https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/imports.js#L71-L95), so we'd have to either change our eslint config or ignore the rule for this line of code. 
2. We're using chalk v5, which moved to ESM, meaning we can't `require('chalk')` without downgrading to v4 or using a pretty hacky-feeling dynamic import workaround. 

Here's basically what that code would look like:

```node
    if (process.env.DEV_LOGIN_LINK && process.env.NODE_ENV === 'development') {
        // eslint-disable-next-line import/no-extraneous-dependencies
        import('chalk').then((chalkModule) => {
            const chalk = chalkModule.default;
            const message = `| Login link generated: ${href} |`;
            console.log(chalk.blue('-'.repeat(message.length)));
            console.log(chalk.blue(message));
            console.log(chalk.blue('-'.repeat(message.length)));
        });
    }
```

Personally, I don't think it's a particular improvement. It also has the small danger of someone copy-pasting code that relies on chalk outside of the `NODE_ENV === 'development'` check and possibly breaking staging/prod. 

If this is the only instance of wanting color output for now, I'd be inclined to leave it as-is. If we start needing this a few places, I'd probably consider promoting chalk to a full dependency. Since this is server-side code, we shouldn't be as worried about bundle size, etc.

Thoughts?

Reply:
Agree with the above – let's leave as-is unless/until there are any compelling use-cases for production.